tests/httplib2test.py

```diff
@@ -7,6 +7,8 @@
 Requires Python 2.4 or later
 """
 
+from __future__ import absolute_import
```

Comment:
I believe they're httplib2's tests.  I don't know why they're included or why we run them.  They were introduced in [this commit](https://github.com/shotgunsoftware/python-api/commit/c0fc67fe3900e5fd13d9f78229fb3cca3a05bc94) and have been untouched since.